priv/migrations/20240910120001_restructure_aexn_transfers.ex

```diff
@@ -0,0 +1,83 @@
+defmodule AeMdw.Migrations.RestructureAexnTransfers do
+  @moduledoc """
+  Reindex AExN transfers to sort them by (txi, idx).
+  """
+  alias AeMdw.Collection
+  alias AeMdw.Db.Model
+  alias AeMdw.Db.State
+  alias AeMdw.Db.DeleteKeysMutation
+  alias AeMdw.Db.WriteMutation
+
+  require Model
+
+  @spec run(State.t(), boolean()) :: {:ok, non_neg_integer()}
+  def run(state, _from_start?) do
+    transfer_mutations =
+      state
+      |> Collection.stream(Model.AexnTransfer, nil)
+      |> Stream.filter(fn {_aexn_type, _from_pk, _txi, to_pk, _value, _log_idx} ->
+        is_binary(to_pk)
```

Comment:
Why is this filter needed? Won't this filter out the `nil` AexnTransfers and not move them to the new format?

Reply:
You're right, added the condition